iati/core/tests/test_rulesets.py

```diff
@@ -123,6 +122,19 @@ def test_ruleset_init_ruleset_multiple_contexts(self):
             assert isinstance(rule, iati.core.Rule)
             assert isinstance(rule, iati.core.RuleAtLeastOne)
 
+    def test_ruleset_is_valid_for_valid_dataset(self):
+        """Check that a Dataset can be validated against the Standard Ruleset."""
+        ruleset = iati.core.default.ruleset()
+        valid_dataset = iati.core.tests.utilities.DATASET_FOR_STANDARD_RULESET_VALID
+        assert ruleset.is_valid_for(valid_dataset)
+
+    # @pytest.mark.parametrize()
+    # def test_ruleset_is_invalid_for_invalid_dataset(self):
```

Comment:
Presuming this is to be implemented later?

Reply:
Presume so - the `[test_]ruleset.py` is merely merged in from the other branches.